src/game_dialog.cc

```diff
@@ -3884,7 +3890,8 @@ int partyMemberCustomizationWindowInit()
             return -1;
         }
 
-        buttonSetCallbacks(_gdialog_buttons[index], _gsound_med_butt_press, _gsound_med_butt_release);
+        // changed index to index + 1 to catch all buttons (otherwise last missing sound)
+        buttonSetCallbacks(_gdialog_buttons[index + 1], _gsound_med_butt_press, _gsound_med_butt_release);
```

Comment:
This seems pretty off kilter.  I think the index here should be the index that we just created `optionButton`

```suggestion
        buttonSetCallbacks(_gdialog_buttons[optionButton], _gsound_med_butt_press, _gsound_med_butt_release);
```